src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Hand.java

```diff
@@ -92,6 +89,7 @@ public boolean isTurnActive() {
     List<Hand> handsStartingWithPreviousWinner = rotateHandsByLastWinner(sortedHands, sortedRounds);
     Optional<Hand> firstHandWhichDidNotPlayCard =
         handsStartingWithPreviousWinner.stream()
+            .filter(Hand::hasCardToPlay)
```

Comment:
so finden wir den spieler, der in der battle round dabei ist, und noch keine karte gespielt hat.
Zum glück haben wir so viele tests